go/integration/gcpkms/gcp_kms_aead_test.go

```diff
@@ -55,7 +55,10 @@ func setupKMS(t *testing.T) {
 	if err != nil {
 		t.Errorf("error setting up gcp client: %v", err)
 	}
-	_, err = g.LoadCredentials(credFile)
+	gcpClient, err := g.LoadCredentials(credFile)
+	if gcpClient == nil {
```

Comment:
We shouldn't check `gcpClient` here, instead, just revert and make `t.Errorf` at line 63 becomes `t.Fatalf`.